### PR TITLE
Improve README on workflows

### PR DESCRIPTION
**Pull request by lldelisle**
Hi there,
I added a part about multi-workflow directories (@bernt-matthias if you can have a look, would be great).
I added a part about the autoupdate and I think we need to change the strategy but I think it is good that it is described here.

### CODE REVIEW

File: workflows/README.md [hunk 1]
```diff
@@ -177,10 +177,41 @@ With a text editor of your choice make this change:
 At this point you can commit the new files and open a pull request.
 If you are encountering difficulties at any point don't hesitate to ask for help on [gitter](https://gitter.im/galaxyproject/iwc).
 
+### Multiple directories or multiple workflows into the same directory
+
+The submitter is free to choose what will match the best its case.
+When multiple workflows are in the same directory, the submitter must know that:
+- The CHANGELOG.md is common and all workflows must constantly have the same version.
+- Workflowhub does not support multiple 'main' workflows in the same directory. One of the workflow will be considred as main while other will be considered as subworkflows. However, this is not an issue for dockstore which will publish each workflow listed in the dockstore file as main.
```
Comment (bernt-matthias) [inline workflows/README.md:185]:
```suggestion
- Workflowhub does not support multiple 'main' workflows in the same directory. One of the workflow will be considred as the main one while all others will be considered as subworkflows. However, this is not an issue for dockstore which will publish each workflow listed in the dockstore file as main.
- ```

Can we define which one will be the main one? Or is this more or less random / alphabetical?

File: workflows/README.md [hunk 2]
```diff
@@ -177,10 +177,41 @@ With a text editor of your choice make this change:
...
+    - `planemo autoupdate` is run in the directory using the [skip list](https://github.com/planemo-autoupdate/autoupdate/blob/main/galaxyproject_iwc_skip_list).
+      - Toolshed updates are checked for all tools used in the workflow. If there are updates, the new version will be used and parameter values will be kept if they stay in the same section with the same name.
+    - If there is a change in the workflow.
+      - The `CHANGELOG.md` is updated using the [python script](https://github.com/planemo-autoupdate/autoupdate/blob/main/pr_text_iwc.py): the log of `planemo autoupdate` is analysed and if there is no PR opened the CHANGELOG file is updated, on top is written the release number and the date. Then under the Automatic update title is written a list of the changes.
+      - If there is a closed PR with the same title. Then everything is cancelled.
+      - If a PR was already opened, the title is updated (but the CHANGELOG has not been updated).
+      - If no PR was opened, a new PR is opened.
+
+### FAQ
+- A workflow has not been updated while it has deprecated tools?
```
Comment (bernt-matthias) [inline workflows/README.md:214]: deprecated -> updated ?

File: workflows/README.md [hunk 3]
```diff
@@ -177,10 +177,41 @@ With a text editor of your choice make this change:
...
+  - Check the [autoupdate actions](https://github.com/planemo-autoupdate/autoupdate/actions). Maybe one workflow raise an error while updating and this prevents other workflows to be updated. Please report this to the matrix channel.
+  - Check that there is no closed PR named 'Updating blabla from xx to xx' corresponding to your workflow. This would prevent any automatic update. If this is the case, leave a comment into this PR so IWC reopen it.
+  - If they are in a multiworkflow directory, this is a known bug and we are working on it. See [this PR](https://github.com/galaxyproject/planemo/pull/1452).
```
Comment (bernt-matthias) [inline workflows/README.md:217]: Not sure if we need to document the bug?

**Review by mvdbeek — APPROVED** [review]

**Comment by mvdbeek** [comment]
Attention: deployment skipped!

https://github.com/galaxyproject/iwc/actions/runs/9695723730

**Pull request merged**